lib/siwapp_web/live/invoices_live/edit.ex

```diff
@@ -185,6 +184,19 @@ defmodule SiwappWeb.InvoicesLive.Edit do
     )
   end
 
+  @spec set_default_series :: list
+  defp set_default_series do
+    Enum.map(Commons.list_series(), fn x -> [key: x.name, value: x.id, selected: x.default] end)
+  end
```

Comment:
The first version received that list as a parameter, but it is used in multiple places, so I decided to include it.

Maybe I must create a `default_series` in the socket, to reuse it, instead of call `Commons.list_series()`.

I'm changing the lambda function.

Reply:
I have refactored a little following the extracting `Commons.list_series()` idea.